Makefile

```
@@ -64,6 +64,10 @@ check-schema:
 check-stream:
 	cargo check --package pallet-stream $(CARGO_FLAGS)
 
+# Rule to check the UNIQUE pallet with all targets
+check-stream:
```

make this `check-unique`

```suggestion
check-unique:
```